### PR TITLE
Remove malformed import statements for clojure 1.10

### PR DESCRIPTION
**Pull request by dpsutton**
This removes import statements that are not valid under 1.10. These are no-ops in versions older so this should not affect anything at all.

Under 1.10 there are some deps that need updating like medley and taoensso.encore but this can happen at a later date.

### CODE REVIEW

**Review by ylebedeva — APPROVED** [review]

**Comment by dpsutton** [comment]
any chance this could be merged and a release cut? Should be a simple no-op except it now passes the stricter ns spec.

**Comment by dpsutton** [comment]
Hi just a friendly reminder about this. I'm fine with forking for our purposes but this is a blocker for Clojure 1.10.1 in our codebase and I'm sure others would benefit as well. The change is trivial and the minimal needed.

**Comment by dpsutton** [comment]
I've forked if anyone needs this patch to upgrade to Clojure 1.10 or 1.10.1

`[dpsutton/saml20-clj "0.1.13"]`

https://github.com/dpsutton/saml20-clj

Changes are _only_ this PR and a coordinate bump to 0.1.13.

**Comment by ylebedeva** [comment]
Hey, I wasn't tracking this PR on assumption you'd merge as an author once it is approved. My apologies for the misunderstanding.

**Comment by ylebedeva** [comment]
Actually thinking about it we should have merged and cut a new point release on March 26 since the release responsibility rests with maintainers anyway. I am sorry. We'll get on it now.

**Comment by dpsutton** [comment]
no worries @jlebedeva . I didn't realize the approval gave me merge permissions. We just merged the fork in today :)

**Comment by dpsutton** [comment]
As soon as i see your most recent build we'll switch back over. Thanks for the helpful lib :)

**Pull request merged**
